netman/adapters/switches/juniper/base.py

```diff
@@ -619,18 +617,25 @@ def node_to_interface(self, interface_node, config):
         else:
             interface.trunk_vlans = vlans
         interface.trunk_native_vlan = value_of(interface_node.xpath("unit/family/ethernet-switching/native-vlan-id"), transformer=int)
-        interface.name = value_of(interface_node.xpath("name"))
```

Comment:
I'd say we need to keep that for backward compatibility for the duration of the deprecation period